plugins/inputs/zookeeper/zookeeper.go

```diff
@@ -22,9 +22,14 @@ import (
 //go:embed sample.conf
 var sampleConfig string
 
-var zookeeperFormatRE = regexp.MustCompile(`^zk_(\w[\w\.\-]*)\s+([\w\.\-]+)`)
+var (
+	zookeeperFormatRE = regexp.MustCompile(`^zk_(\w[\w\.\-]*)\s+([\w\.\-]+)`)
+)
+
+const (
+	defaultTimeout = 5 * time.Second
+)
```

Comment:
I hate those definitions! They add three lines without any benefit instead of just using the value at the corresponding location...

Reply:
Just use this value